torch/testing/_internal/common_utils.py

```diff
@@ -777,27 +777,65 @@ def safeCoalesce(self, t):
 
         return tg
 
-    def assertEqual(self, x, y, prec=None, message='', allow_inf=False, exact_dtype=None):
+    # Some analysis of tolerance by logging tests from test_torch.py can be found
+    # in https://github.com/pytorch/pytorch/pull/32538.
+    dtype_precisions = {
+        torch.half: 0.001,
+        torch.bfloat16: 0.016,
+        torch.float: 1.3e-6,
+        torch.double: 5e-7,
+        torch.int: 0
```

Comment:
Maybe you want to just have all the integral types be zero, and not include them in this map?

Reply:
I actually remapped all integral types to torch.int, but that's probably confusing, so I'll fix that.